dts/bindings/dma/dmamux-controller.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2019, STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+# Common fields for DMAMUX controllers
+
+include: base.yaml
+
+bus: dmamux
+
+properties:
+    label:
+      required: true
+
+    "#dma-cells":
+      type: int
+      required: false
+      description: Number of items to expect in a DMA specifier
+
+    dma-masters:
+      type: phandle-array
+      description: Array of phandles to the DMA controllers the router can direct the signal to.
+
+    dma-channels:
+      type: int
+      required: true
+      description: Number of DMAMUX output request channels supported by the controller
+
+    dma-generators:
```

Comment:
removed if not used

Reply:
yes, for stm32wb55 the request ID cannot exceed 40 = Number of DMAMUX peripheral request inputs + Number of DMAMUX request generator channels
no, for stm32g4 the request ID cannot exceed 115 =  Number of DMAMUX peripheral request inputs

Reply:
problem is that I can't see where it is used..

Reply:
dmamux gen_nb  is compared to the given request ID : request ID must not exceed Number of DMAMUX request generator channels + Number of DMAMUX peripheral request inputs (for stm32wb